be/src/exprs/cast_expr_json.cpp

```diff
@@ -208,4 +208,13 @@ StatusOr<ColumnPtr> cast_nested_to_json(const ColumnPtr& column) {
     return column_builder.build(false);
 }
 
+StatusOr<std::string> cast_type_to_json_str(const ColumnPtr& column, int idx) {
```

Comment:
why not use `Datum` as argument ?

Reply:
just for the convenience of calling cast_datum_to_json(const ColumnPtr& col, int row, const std::string& name, vpack::Builder* builder)